doc/containers.md

```diff
@@ -172,7 +172,7 @@ host\_name              | string    | randomly assigned | no        | bridged, p
 hwaddr                  | string    | randomly assigned | no        | all                           | -             | The MAC address of the new interface
 mtu                     | integer   | parent MTU        | no        | all                           | -             | The MTU of the new interface
 parent                  | string    | -                 | yes       | physical, bridged, macvlan    | -             | The name of the host device or bridge
-vlan                    | integer   | -                 | no        | macvlan                       | network\_vlan | The VLAN ID to attach to
+vlan                    | integer   | -                 | no        | macvlan, physical             | network\_vlan | The VLAN ID to attach to
```

Comment:
And a new paragraph should be added below that explains the use of VLANs with LXD. Making it clear that if using the vlan property with a physical device, only a single container can use the VLAN and that if one wants more than one container to use a given VLAN, they should instead use macvlan for all of them.